topic/timeseries/requirements.txt

```diff
@@ -1,3 +1,4 @@
-pycaret>=3.0
-refinitiv-data
 crate[sqlalchemy]==0.34.0
+refinitiv-data<1.7
+pycaret>=3.0,<3.4
+pydantic<2
```

Comment:
Just re-ordering and limiting, to let Dependabot drive the version bumps, in order to let CI jobs run frequently on this folder, even without other interactions.